Lua/Plugins/py.lua

```diff
@@ -0,0 +1,80 @@
+--- Highlight keywords
+highlight("import", "reserved")
+highlight("from", "reserved")
+highlight("while", "reserved")
+highlight("if", "reserved")
+highlight("else", "reserved")
+highlight("elif", "reserved")
+
+highlight("false", "binary")
+highlight("true", "binary")
```

Comment:
"False" and "True" are capitalized in python

(Apologies, I'm not quite versed in the acts of the Hub so I'm unsure as to how I'd propose changes, how RealPacket did)

Reply:
You can do <kbd>CTRL</kbd>+<kbd>G</kbd>.